service/core/client/src/main/scala/com/lightbend/lagom/internal/client/CircuitBreakerMetricsProviderImpl.scala

```diff
@@ -12,11 +12,12 @@ import com.codahale.metrics._
 import com.lightbend.lagom.internal.spi.{ CircuitBreakerMetrics, CircuitBreakerMetricsProvider }
 import play.api.inject.{ Binding, Injector, Module }
 import play.api.{ Configuration, Environment, Logger }
+import com.lightbend.lagom.client.CircuitBreakersImpl
 
 class CircuitBreakerModule extends Module {
   override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] = {
     Seq(
-      bind[CircuitBreakers].toSelf,
+      bind[com.lightbend.lagom.client.CircuitBreakers].to[CircuitBreakersImpl],
```

Comment:
Here I had to full-qualify the new `CircuitBreakers` because the module itself is under `com.lightbend.lagom.internal.client`.

I'm wondering if we should move this module under `com.lightbend.lagom.client` as well.